ml_pipeline_engine/node/node.py

```diff
@@ -67,7 +69,7 @@ def run_node_default(node: NodeLike[NodeResultT], **kwargs: t.Any) -> t.Type[Nod
     return get_instance(node).get_default(**kwargs)
 
 
-async def run_node(node: NodeLike[NodeResultT], *args: t.Any, **kwargs: t.Any) -> t.Type[NodeResultT]:
+async def run_node(node: NodeLike[NodeResultT], *args: t.Any, node_id: NodeId, **kwargs: t.Any) -> t.Type[NodeResultT]:
     """
     Функция для запуска узла.
```

Comment:
We've changed the function, we can do a translation & update of the dock.

Reply:
Fixed